schema/places/defs.yaml

```diff
@@ -8,3 +8,9 @@ description: Common schema definitions for places theme
       type: string
       minLength: 1
       pattern: ^[a-z0-9]+(_[a-z0-9]+)*$
+    basic_level_category:
```

Comment:
My suggestion would be call it `basic_category` and document it as "Basic level category"